beacon-chain/utils/shuffle_test.go

```diff
@@ -72,3 +72,20 @@ func TestSplitIndices_OK(t *testing.T) {
 		}
 	}
 }
+
+func TestSplitOffset_OK(t *testing.T) {
```

Comment:
Can you add a test that tests splitoffset only? This test seems to depend on SplitIndicies

Reply:
addressed here https://github.com/prysmaticlabs/prysm/pull/2365/commits/d0e4a4772ab0fffa2d7e632e2cad9cf0f3030af3